src/Fixer/ControlStructure/NoUselessElseFixer.php

```diff
@@ -198,6 +229,35 @@ private function fixEmptyElse(Tokens $tokens, $index)
      */
     private function clearElse(Tokens $tokens, $index)
```

Comment:
I thought about it: converting a `elseif` into a `if` is basically a `else` removal so I decided to keep on method. But all the new logic is in a specific `if` so using a new method instead makes sense too.